.github/workflows/check-patch.yml

```diff
@@ -0,0 +1,105 @@
+name: Check patch
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build-el8:
+
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/centos/centos:stream8
+
+    steps:
+    - name: prepare env
+      run: |
+           mkdir -p ${PWD}/tmp.repos/BUILD
+           yum install -y autoconf automake gettext-devel git systemd make git rpm-build
+
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: autoreconf
+      run: autoreconf -ivf
+
+    - name: configure
+      run: ./configure
+
+    - name: run distcheck
+      run: make distcheck
```

Comment:
nit: do we know how many processors we have, so we can run make in multiple threads?

Reply:
no, but for this specific package it won't provide much gain.

Reply:
I can set -j without arguments and just not limit the number of parallel jobs. But I believe that for distcheck here there won't be improvements.